openpype/pipeline/colorspace.py

```diff
@@ -221,8 +323,46 @@ def get_data_subprocess(config_path, data_type):
             "config", data_type,
             "--in_path", config_path,
             "--out_path", tmp_json_path
+        ]
+        log.info("Executing: {}".format(" ".join(args)))
+
+        process_kwargs = {
+            "logger": log
+        }
+
+        run_openpype_process(*args, **process_kwargs)
+
+        # return all colorspaces
+        return_json_data = open(tmp_json_path).read()
+        return json.loads(return_json_data)
```

Comment:
This should be:
```suggestion
        # return all colorspaces
        with open(tmp_json_path, "r") as f:
            return json.load(f)
```
Also further down in next function.